### PR TITLE
add English translation of code part for word2vec

### PR DESCRIPTION
**Pull request by helinwang**

### CODE REVIEW

File: word2vec/README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ CBOW的好处是对上下文词语的分布在词向量上进行了平滑，去
 
 ### 数据介绍
 
-本教程使用Penn Tree Bank (PTB)数据集。PTB数据集较小，训练速度快，应用于Mikolov的公开语言模型训练工具\[[2](#参考文献)\]中。其统计情况如下：
+本教程使用Penn Tree Bank （PTB）（经Tomas Mikolov预处理过的版本）数据集。PTB数据集较小，训练速度快，应用于Mikolov的公开语言模型训练工具\[[2](#参考文献)\]中。其统计情况如下：
```
Comment (wangkuiyi) [inline word2vec/README.md]: Penn Treebank

**Review by wangkuiyi — APPROVED** [review]

File: word2vec/README.md (hunk 1 above)
Comment (helinwang) [reply word2vec/README.md]: Done.

File: word2vec/index.en.html [hunk 2]
```diff
@@ -191,19 +191,257 @@
...
+nextword = paddle.layer.data(
+    name="fifthw", type=paddle.data_type.integer_value(dict_size))
+
+Efirst = wordemb(firstword)
+Esecond = wordemb(secondword)
+Ethird = wordemb(thirdword)
+Efourth = wordemb(fourthword)
+```
+
+- Concat n-1 word embedding vectors into a single feature vector.
```
Comment (gangliao) [inline word2vec/index.en.html]: Concat -> Concatenate

Comment (helinwang) [reply word2vec/index.en.html]: Done.

**Pull request merged**